code/modules/cargo/exports/materials.dm

```diff
@@ -15,46 +16,97 @@
 
 	return M.amount
 
-// Materials. Nothing but Phoron is really worth selling. Better leave it all to RnD.
+// Materials. As of writing in 2024, this list was small, and most everything was effected by price elasticity. Now, it's been rebalanced. Price elasticity was removed,
+// and more materials now have values. These materials are mainly concerend with mining.
+
 
 /datum/export/material/diamond
-	cost = 100
+	cost = 20
 	material_id = "diamond"
 	message = "diamond sheets"
 
 /datum/export/material/phoron
 	cost = 175
-	k_elasticity = 0
 	material_id = "phoron"
 	message = "phoron sheets"
 
 /datum/export/material/uranium
-	cost = 10
+	cost = 8
 	material_id = "uranium"
 	message = "uranium sheets"
 
 /datum/export/material/gold
-	cost = 25
+	cost = 12
 	material_id = "gold"
 	message = "gold sheets"
 
 /datum/export/material/silver
-	cost = 10
+	cost = 8
 	material_id = "silver"
 	message = "silver sheets"
 
 /datum/export/material/platinum
-	cost = 20
+	cost = 12
 	material_id = "platinum"
 	message = "platinum sheets"
 
+/datum/export/material/osmium
+	cost = 24
```

Comment:
more than diamond?

Reply:
Only more than diamond because smelting osmium is equal to two platinum ore. So you're getting the same amount of value regardless of compressed vs smelted